internal/batches/docker/info.go

```diff
@@ -35,6 +34,13 @@ func CurrentContext(ctx context.Context) (string, error) {
 	return name, nil
 }
 
+type Info struct {
+	Host struct {
+		CPUs int `json:"cpus"`
+	} `json:"host"`
+	NCPU int `json:"NCPU"`
```

Comment:
Can you add some doc strings here explaining that only some values are used by podman and Docker? Docker doesn't have `Host`, but it has `NCPU`, right?

Reply:
Exactly, added a comment